ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/models/tfgrid_solutions_network_1.toml

```diff
@@ -3,3 +3,4 @@ name** = ""
 rid = ""
 form_info = (dict)
 explorer = ""
+parent_network = ""
```

Comment:
shouldnt this be an int or we need to add int() around it in the other PR

Reply:
@grimpy  the int default value can't be empty in bcdb, that is why I added it as a string. Regarding the other PR, the reservation_get(..) and reservation_cancel(..) both can take the id as a string or int so we don't need to add int()